contracts/StrategyConvexCurveIronBankLP.sol

```diff
@@ -208,14 +204,14 @@ contract StrategyConvexCurveIronBankLP is BaseStrategy {
                 want.balanceOf(address(this))
             );
         }
-
-        return (_profit, _loss, _debtPayment);
```

Comment:
when your method defines return variable with names, you don't need to return them. That's done automagically.